src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json

```diff
@@ -3,6 +3,7 @@
     "name": "Hans Muster",
     "phone": "9482424",
     "email": "invalid@email!3e",
-    "address": "4th street"
+    "address": "4th street",
+    "quarantineStatus": "tru"
```

Comment:
Minor spelling error "true"

Reply:
Spelling error is intentional for the invalid person.